src/main/java/com/hubspot/jinjava/tree/TagNode.java

```diff
@@ -84,4 +86,17 @@ public Tag getTag() {
     return tag;
   }
 
+
+  private String reconstructImage() {
+    StringBuilder builder = new StringBuilder().append(master.getImage());
+
+    for (Node n : getChildren()) {
+      builder.append(n.getMaster().getImage());
+    }
+
+    builder.append("{% ").append(getEndName()). append(" %}");
```

Comment:
can you use/create constants for `{%` and `%}`? People have requested to make these configurable and I'd like to consolidate them.

Reply:
Sure